### PR TITLE
Dockerfile: using upstream AUFS bug workaround

### PR DESCRIPTION
**Pull request by bryanlatten**
Also restructured tests to not be in the root folder

### CODE REVIEW

File: container/root/clean.sh [hunk 1]
```diff
@@ -24,4 +24,4 @@ apt-get autoclean -y && \
 apt-get autoremove -y && \
 rm -rf /var/lib/{cache,log}/ && \
 rm -rf /var/lib/apt/lists/ && \
-rm -rf /tmp/* /var/tmp/*
```
Comment (bryanlatten) [inline container/root/clean.sh:27]: required to allow the aufs hack to survive :)

**Comment by digitalyuki** [comment]
LGTM

File: container/root/app/public/index.php [hunk 2]
```diff
@@ -8,3 +8,16 @@
 fwrite( $stderr, 'Using STDERR pipe for output' );
 
 phpinfo();
+
+// Use the below form to test uploads
+?>
+
+<form method="post" action="/" enctype="multipart/form-data">
```
Comment (bryanlatten) [inline container/root/app/public/index.php:15]: adding a quick and dirty self-posting form to allow basic-level testing that POST uploads to the server work :)

**Comment by marknotfound** [comment]
LGTM

**Pull request merged**